src/Pomodoro.Web/wwwroot/css/app.css

```
@@ -3252,3 +3252,15 @@ textarea.tep-input {
         padding: 9px 11px;
     }
 }
+
+.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)